### PR TITLE
Map branch

### PR DESCRIPTION
**Pull request by OlyaMilch**
Create 2 new classes: NetworkIngredient with method json and MapNetwork with method map.
These methods are appli to 'get_ingredients' and 'get_ingredient_by_id'

### CODE REVIEW

File: ingredients.py [hunk 1]
```diff
@@ -20,18 +21,21 @@ def __repr__(self):
 @app.route('/ingredients', methods=['GET'])  # Смотрим весь список
 def get_ingredients():
     ingredients = Ingredient.query.all()
-    return jsonify(
-        [{'id_i': ingr.id_i, 'title': ingr.title, 'category': ingr.category} for ingr in
-         ingredients]), 200
+    network_ingredient = [
+        MapNetwork.map_ingredient(ingr).__dict__  # без дикта не очень красивый вывод
+        for ingr in ingredients
```
Comment (Makentoshe) [inline ingredients.py:26]: Ingredient

File: ingredients.py [hunk 2]
```diff
@@ -53,3 +57,20 @@ def delete_ingredient(id_i: int):
...
+        self.id_i = id_i
+        self.title = title
+        self.category = category
+
+    def to_json(self):  # просто селф, т.к. мы УЖЕ внутри класса NetworkIngredient
+        data = json.dumps(self.__dict__)  # дикт возвращает все по ключам
+        return data
+
+
+class MapNetwork:
```
Comment (Makentoshe) [inline ingredients.py:73]: Rename to like IngredientToNetworkIngredientMapper

File: ingredients.py [hunk 3]
```diff
@@ -20,18 +21,21 @@ def __repr__(self):
...
+    ]  # перебрали элементы, смаппили их и в json перевели
+    return json.dumps(network_ingredient), 200  # вывод на одной строке в браузере благодаря json
 
 
 # Получаем информацию по айди
 @app.route('/ingredient/<int:id_i>', methods=['GET'])
 def get_ingredient_by_id(id_i):
     ingredient = Ingredient.query.get(id_i)
-    if ingredient == None:
+    if ingredient == None:  # сначала проверка, потом маппер
```
Comment (Pelmen323) [inline ingredients.py:35]: The comparison to "None" should use "is" instead of "=="

File: ingredients.py [hunk 4]
```diff
@@ -2,11 +2,12 @@
 from flask_sqlalchemy import SQLAlchemy
```
Comment (Pelmen323) [inline ingredients.py:2]: Clean up imports

File: ingredients.py (hunk 1 above)
Comment (OlyaMilch) [reply ingredients.py:26]: all set

File: ingredients.py (hunk 2 above)
Comment (OlyaMilch) [reply ingredients.py:73]: fixed

File: ingredients.py [hunk 5]
```diff
@@ -53,3 +57,20 @@ def delete_ingredient(id_i: int):
...
+        self.category = category
+
+    def to_json(self):  # просто селф, т.к. мы УЖЕ внутри класса NetworkIngredient
+        data = json.dumps(self.__dict__)  # дикт возвращает все по ключам
+        return data
+
+
+class MapNetwork:
+    @classmethod  # не cоздаем объект от класса, а вызываем объект от класса
+    def map_ingredient(cls, ingredient: Ingredient):  # cls-обозначает класс, а не объект
```
[the remark this answers is not in the document]
Comment (OlyaMilch) [reply ingredients.py:75]: fixed

File: ingredients.py (hunk 3 above)
Comment (OlyaMilch) [reply ingredients.py:35]: fixed

File: ingredients.py (hunk 4 above)
Comment (OlyaMilch) [reply ingredients.py:2]: fixed

**Pull request merged**